src/conversions.jl

```diff
@@ -359,12 +359,11 @@ and stores the `ClosedInterval` to `n` and `m`, plus the original matrix in a Tu
 """
 function convert_arguments(sl::SurfaceLike, data::AbstractMatrix)
     n, m = Float32.(size(data))
-    convert_arguments(sl, 0f0 .. n, 0f0 .. m, el32convert(data))
+    convert_arguments(sl, firstindex(data, 1) .. lastindex(data, 1), firstindex(data, 2) .. lastindex(data, 2), el32convert(collect(data)))
```

Comment:
Should these also be converted to `Float32`?

Reply:
Eh debatable...might be simpler to leave it as is, since we'll try to not convert to float32 as much in the future.  Plus, I don't think it's crazy inefficient to do things this way?

Reply:
I don't know, I just noticed you do this in the other method so they aren't consistent